index.bs

```diff
@@ -1564,6 +1565,90 @@ invoked, must run these steps:
 
 </div>
 
+# Drag and Drop Integration # {#drag-and-drop}
+
+During a <em>drag-and-drop operation</em>, dragged file and
+directory items are associated with [=file entries=] and [=directory entries=] 
+respectively. Each [=/entry=] is stored as an element in the <l spec=html>{{DragEvent|DragEvent's}}</l>
+<l spec=html>[=drag data store=] </l> and can be accessed using {{DataTransferItem/getAsFileSystemEntry()}} on
+the entry's associated <l spec=html>{{DataTransferItem}}</l>.
+
+## The getAsFileSystemEntry() method ## {#api-getasfilesystementry}
+
+{{DataTransferItem/getAsFileSystemEntry()}} will return a {{FileSystemFileHandle}} object if
+the dragged item is file and a {{FileSystemDirectoryHandle}} object if the dragged item
+is a directory.
+
+Dragging and dropping a file/directory will grant the same permissions as if they were
+accessed via the [=native file system handle factories=], with the exception that [=too sensitive or dangerous|sensitive directories=]
+are not blocked via drag and drop.
+
+Issue: Discuss blocking when dragging forbidden sensitive directories onto the web. For
+now these directories can be accessed through drag and drop.
+
+Issue: Eventually there should be discussion about integrating drag and drop operations
+with `<input type='file'>` elements, but for now, there is no interaction.
+
+<xmp class=idl>
+partial interface DataTransferItem {
+    FileSystemHandle? getAsFileSystemEntry();
```

Comment:
It's kind of confusing how sometimes we're talking about handles and sometimes about entries, but this method should probably actually be getAsFileSystemHandle, given that that is the type that is being returned.

Reply:
Done.